### ISSUE TITLE
Design the dashboard view for multiple APD types

### ISSUE THREAD

**Issue by beparticular** [CMSgov/eAPD#2456]
With multiple submissions and multiple APD types, APD names will have to not all be the same as they currently are in the system.
States should be able to easily view identifying info/data listed in #2379, date in issues that branch from #1823 + anything else? 

Current dashboard view for info shown to identify an APD
![Screen Shot 2020-08-18 at 2.57.02 PM.png](https://images.zenhubusercontent.com/591f3b9351aa2c30577ecf92/33884010-ddb7-479a-bfa3-43353d413ae1)

### This task is done when...
 - [ ] Sketches or wireframes are created to discuss ideas (fidelity level at discretion of design)
 - [ ] Mock ups for the UI are created 
 - [ ] Design is approved by product

**Comment by jeromeleecms** [comment]
Maybe a separate issue, but we should also consider an option for potentially archiving/inactivating/hiding APDs from this view. Multiple submissions can potentially make this list super long.

**Comment by jjames521** [comment]
https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2020?node-id=787%3A8

**Comment by jjames521** [comment]
This Figma link (below) via the "gold star" start of flow for conceptual dashboard designs.

This satisfies the following:

**Re-thought out display of list information.**
How do the print and the dashboard match for sharing information outside of the system then returning to edit/correct, etc?
Long "names" or identifying info, when does it need to wrap

**Inclusion of review or completion status bars, and a new Project, APD hierarchy display.**
Review or Completion Status
Projects vs APDs vs Activities vs Types


**The solution for this would be ability to activate through the check box system to archive, etc.**
Consider archiving/inactivating/hiding APDs (this could be a separate issue if it needs to be)
Archiving


https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2020?node-id=1251%3A29

**Comment by beparticular** [comment]
@akuas from our sync discussion - 

Things to think about at this time
how would we show APDs connect (p-APD, i-APD, o-APD, updates) 
how would we allow people to create a connected APD 
how to differentiate between starting a new APD and starting a connected APD

**Comment by SGilliamA1M** [comment]
@jeromeleecms and @stephanieboydcms - the following [Dashboard](https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4314%3A19558) updates are available for your review and feedback:
- Inclusion of "APD Type"
- Inclusion of  "APD Funding Source"
- Inclusion of "Last Edited and Created"

**Comment by stephanieboydcms** [comment]
Thank you for an opportunity to review.  Just two minor edits left on Figma.  @SGilliamA1M & @jeromeleecms

**Comment by SGilliamA1M** [comment]
@jeromeleecms and @stephanieboydcms - thank you both for providing feedback on the Dashboard updates. Here are some closing notes to capture changes:

1. Remove intro/help text copy from the Dashboard.
2. Group APDs by Type (i.e: MMIS, HITECH, etc.) <- Will revisit post-Pilot launch for alternate grouping (_i.e: active or complete_)
3. Data fields should capture: APD Name, APD Type, FFY, Last edited, Created, Last edited author

**Comment by jeromeleecms** [comment]
Sounds good - going to reassign to Dev for ticket creation!

**Comment by thetif** [comment]
@jeromeleecms for this ticket, we need to discuss how we want to record users editing the APD. Are we locking the APD so only one use can edit at a time? Do we record each edit? All of the edits while it is locked? If we just update it on every edit if you are watching the dashboard (once it's hooked up to websockets) it's just going to be constantly updating.

**Comment by jeromeleecms** [comment]
Good point. The initial plan is we will be locking it down so only one user can edit it at one time. My thought was if it's locked, the prospective user can see who might be in it. If you think this may be too confusing or too distracting for this iteration, I'm open to removing it for speed of delivery. @SGilliamA1M?

**Comment by thetif** [comment]
dev ticket #4053

**Comment by jeromeleecms** [comment]
@mirano-darren As a reminder, we are removing "Last Edited" by in this design to simplify and speed rollout of this (per our parking lot discussion)

**Comment by jeromeleecms** [comment]
Dev ticket reviewed. Good to close.
